src/main/java/io/jawg/osmcontributor/flickr/rest/asynctask/GetFlickrPhotos.java

```diff
@@ -62,21 +64,34 @@
 
     private Integer nbPage = 1;
 
-    public GetFlickrPhotos(Double longitude, Double latitude, Flickr flickr, Integer limitPerPage, Integer nbPage) {
+    private Poi featurePoi;
+
+    public GetFlickrPhotos(Double longitude, Double latitude, Flickr flickr, Integer limitPerPage, Integer nbPage, Poi featurePoi) {
         this.longitude = longitude;
         this.latitude = latitude;
         this.flickr = flickr;
         this.limitPerPage = limitPerPage;
         this.nbPage = nbPage;
+        this.featurePoi = featurePoi;
     }
 
     @Override
     protected List<List<Size>> doInBackground(Void... params) {
+        //Create search tags list
+        ArrayList<String> searchTags = new ArrayList<String>(Arrays.asList(TAGS));
```

Comment:
We can't initialize an array directly within an Arrays.asList call. However, it can be done this way:
```
    private static final String[] TAGS_ARRAY = { "openstreetmap" };
    private static final List<String> TAGS = Arrays.asList(TAGS_ARRAY);
```